filament/CMakeLists.txt

```diff
@@ -191,6 +191,11 @@ list(APPEND SRCS ${RESGEN_SOURCE})
 # Configuration
 # ==================================================================================================
 
+# whether we're building for mobile -- this can affect some default quality settings
+if (IS_MOBILE_TARGET)
+    add_definitions(-DTARGET_MOBILE=1)
```

Comment:
Can you preface this definition with `FILAMENT_`?

Reply:
good idea, I went with the same define we use in shaders, but really this one should've had the prefix too.